content/cybersecurity/concepts/nmap/terms/udp-scan/udp-scan.md

```diff
@@ -0,0 +1,58 @@
+---
+Title: 'UDP Scan' 
+Description: 'Scans (only) the UDP Ports of a target, which aren't scanned in a regular scan. Can be combined with any TCP scan' 
+Subjects: 
+  - 'Computer Science'
+  - 'Networking'
+Tags: 
+  - 'Networking'
+  - 'Cybersecurity'
+  - 'Ethical Hacking'
+CatalogContent: 
+  - 'introduction-to-cybersecurity'
+  - 'paths/scan-systems-with-nmap'
+---
+
+An **UDP Scan** scans (only) the UDP Ports of a target, while the [regular scan](https://www.codecademy.com/resources/docs/cybersecurity/nmap/nmap-scan) only scans the TCP Ports. It is performed by using the -sU option and can be combined with any TCP scan type, for example SYN scan (-sS) to scan UDP and TCP Ports in the same run. UDP Scan is normally slower and more difficult than a TCP scan.
```

Comment:
```suggestion
An **UDP Scan** scans (only) the UDP Ports of a target, while the [regular scan](https://www.codecademy.com/resources/docs/cybersecurity/nmap/nmap-scan) only scans the TCP Ports. It is performed by using the `-sU` option and can be combined with any TCP scan type, for example SYN scan (`-sS`) to scan UDP and TCP Ports in the same run. UDP Scan is normally slower and more difficult than a TCP scan.
```

Reply:
I have a question about the md file: the metadata at the beginning isn't displayed correctly when opening https://github.com/jansoth/CodecademyDocs/blob/NmapUDPScan/content/cybersecurity/concepts/nmap/terms/udp-scan/udp-scan.md (as shown in the image).
I can't find the mistake when comparing the source to for example the 'agressive scan' source in the md file

![grafik](https://user-images.githubusercontent.com/25724976/208101233-60ad3871-b0b3-4767-b34c-09cad569661b.png)